projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -34,6 +34,26 @@ public function register_routes() {
 			)
 		);
 
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/integration-status/(?P<slug>[\w-]+)',
+			array(
+				'methods'             => \WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_integration_status' ),
+				'permission_callback' => array( $this, 'get_items_permissions_check' ),
+				'args'                => array(
+					'slug' => array(
+						'type'              => 'string',
+						'required'          => true,
+						'sanitize_callback' => 'sanitize_text_field',
+						'validate_callback' => function ( $param ) {
+							return preg_match( '/^[\w-]+$/', $param );
```

Comment:
What does this do? Lets leave a comment here.

Reply:
This pre-validates that the slug contains only letters, numbers, underscores, and hyphens; has at least one character; does no contain spaces or any other special characters.